src/push-notifications.js

```diff
@@ -71,6 +73,56 @@ export class PushNotifications {
 
     return doRequest("POST", path, { token });
   }
+
+  _initDb(dbName) {
+    var request = indexedDB.open(dbName);
+
+    request.onerror = function(event) {
+      console.log(`Database error: " + ${event.target.errorCode}`);
```

Comment:
What is the best way to communicate errors to the users?
What should we do if this operation fails? Retry?

Reply:
You don't need the `" +` when using string interpolation :)

Reply:
And good question about handling failed operations, we should have a think